tc-messaging/src/main/java/com/tc/entity/NetworkVoltronEntityMessageImpl.java

```diff
@@ -150,12 +159,21 @@ protected boolean hydrateValue(byte name) throws IOException {
     this.extendedData = getBytesArray();
     this.requiresReplication = getBooleanValue();
     this.oldestTransactionPending = new TransactionID(getLongValue());
+    
+    try {
+      if (this.type == Type.INVOKE_ACTION) {
+        MessageCodec<EntityMessage, EntityResponse> codec = supplier.getMessageCodec(this.entityDescriptor.getEntityID());
+        this.message = codec.decodeMessage(extendedData);
+      }
+    } catch (MessageCodecException exception) {
+//  swallow it
```

Comment:
You should flesh out more of the rationale behind why we can just swallow the exception.  I assume that the reason is that this is best-efforts, anyway, since the original use of the message field was a corner-case so we already have the machinery in place to attempt the decode, later on, if we don't run it here.